### ISSUE TITLE
Boot from GPT system disk not working

### ISSUE THREAD

**Issue by lellis1936** [Drive-Trust-Alliance/sedutil#412]
I recently converted my system disk (which also already was using SEDUTIL) to GPT using mbr2gpt.  Everything went well.

After that I installed the UEFI64 PBA image (I had been previous using the lean BIOS PBA).  The UEFI boot of the PBA environment worked fine.  But the chainloading of my Windows OS did not.  The following sequence of messages occurred:

**About to Chainload hd0,0 Booting...**

**Reboot and Select proper Boot device**
**or Insert Boot Media in selected Boot device and press a key**

If, at this point, I press a key, the cursor drops down a line, continues to blink, but nothing else happens.  If instead I use CTRL-ALT-DELETE, the reboot occurs, and the OS loads fine.

Any ideas about if, and how, I can get this working?

**Comment by lellis1936** [comment]
I justed wanted to add that, since this OS drive was converted to GPT after OS installation, the EFI system partition is not in the ususal postiion (first partition) of the drive.  It is the third partition of the drive.  

I don't know how chainloading works but it makes me wonder if  **hd0,0** is correct, and whether and how I can change that.

**Comment by lellis1936** [comment]
I discovered the answer to this dilemma on my own through experimentation.

What was happening is that my BIOS (which is quite old), was doing an MBR boot of the PBA, not a UEFI one.  My BIOS requires a UEFI boot menu option to exist in most boot scenarios.  There wasn't one for the PBA environment, so the MBR boot was used.

I used EasyUEFI bootable to add a UEFI boot item for the SEDUTIL UEFI PBA.  After that, the system boots in UEFI mode as it should.  It does do a BIOS post after the disks are unlocked, so in my scenario it's not a huge improvement over the MBR flavor, although it IS nice not to have to use the three-finger CTRL-ALT-DEL sequence.
